packages/tools/fetch-tool/package.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"main": "dist/fluidFetch.js",
+	"main": "dist/index.js",
```

Comment:
Are you sure abou this one?

Reply:
oops missed this, not sure how that happened